### PR TITLE
LibWasm: Implement a lot more SIMD instructions

### PR DESCRIPTION
**Pull request by alimpfard**
5.4k more passing tests :P
`Tests:       18762 failed, 1735 skipped, 31043 passed, 51540 total`

### CODE REVIEW

File: Userland/Utilities/wasm.cpp [hunk 1]
```diff
@@ -38,6 +39,201 @@ static void sigint_handler(int)
...
+        } else if (digit >= 'a' && digit <= 'f') {
+            digit_val = 10 + (digit - 'a');
+        } else if (digit >= 'A' && digit <= 'F') {
+            digit_val = 10 + (digit - 'A');
+        } else {
+            return {};
+        }
+
+        value = (value << 4) + digit_val;
+    }
```
Comment (gmta) [inline Userland/Utilities/wasm.cpp:86]: This all reads very similar to `AK::decode_hex(_digit)`, can't we reuse any of that?

Comment (alimpfard) [reply Userland/Utilities/wasm.cpp:86]: None of the hex readers I looked at could handle u128, I'll take a look at that one too.

Comment (alimpfard) [reply Userland/Utilities/wasm.cpp:86]: Oh that decided into an allocated buffer, producing a u128 from that is probably more code tbh

Comment (gmta) [reply Userland/Utilities/wasm.cpp:86]: Maybe just decode_hex_digit then?

Comment (alimpfard) [reply Userland/Utilities/wasm.cpp:86]: Sure thing.

**Pull request merged**